src/payment.js

```diff
@@ -0,0 +1,174 @@
+const cart = require('./cart');
+const cardApi = require('./card');
+const { isFunction, vaultRequest } = require('./utils');
+
+function methods(request) {
+  return {
+    async createElements(params = {}, checkoutId) {
+      const gateways = await vaultRequest('get', '/gateways');
+      if (gateways.error) {
+        throw new Error(gateways.error);
+      }
+      await render(request, params, checkoutId, gateways);
+    },
+
+    async create(orderId, amount, method, source) {
+      return request('post', `/payments`, {
```

Comment:
@vstudenichnik-insoft we shouldn't need this after all -- instead of calling `payment.create` we should update the current cart with relevant payment details

billing.card.token
billing.paypal
etc

Reply:
Done